daos/rinkeby/necdaocurvelabs.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "necDAO",
+  "Avatar": "0x331dDC7Ed27451b5B4d25A41e8dF53cFF456DcA9",
+  "DAOToken": "0x6939C5672b575C2fa4E3393D1706c7A097219805",
+  "Reputation": "0x814d49253d595Ec7341e7307d2CB4964Ab9655F6",
+  "Controller": "0x5bdD6bdb6D35920DC1DeD2101Cb9b1D39DCEa92D",
+  "Schemes": [
+    {
+      "name": "UniswapProxy",
+      "alias": "UniswapProxy",
+      "address": "0x50AE25C926176D71F41A27Fe97fa4A7c31207131",
+      "arcVersion": "0.0.1-rc.44"
+    },
+    {
+      "name": "GenericScheme",
+      "alias": "Uniswap",
+      "address": "0xc3524072aa711f84d1Db93B1cdad68be176162bd",
```

Comment:
it will be nice to verify [this](https://rinkeby.etherscan.io/address/0xc3524072aa711f84d1Db93B1cdad68be176162bd#code) also

Reply:
This is a `GenericScheme` contract. I can't verify it because it has been automatically deployed through the `@daostack/migration` package and thus I don't have access to the exact `constructor` params - notably the voting params hash.

Reply:
ok. I verified it :)